docs/source/tutorials/packaging_output_models.md

```diff
@@ -0,0 +1,61 @@
+# Packaing Olive artifacts
+
+## What is Olive Packaging
+Olive will output multiple candidate models based on metrics priority ranks. It also can package output artifacts when the user required. Olive packaging can be used in different scenarios. There are 2 different packaging types: `General` and `AzureML`.
+
```

Comment:
Since AzureML packaging is not included in current PR, just say that currently only Zipfile package is supported.

Reply:
Done.